common/src/main/java/com/scottlogic/deg/common/ValidationException.java

```diff
@@ -20,7 +20,7 @@
 import java.util.List;
 
 public class ValidationException extends RuntimeException {
-    public final List<String> errorMessages;
+    public final transient List<String> errorMessages;
```

Comment:
I don't think we should make this `transient`. It implies we have thought about serialising this class, or other areas.